templates/base.html

```diff
@@ -90,12 +90,6 @@
         </div>
         <!-- Footer -->
         <footer class="page-footer font-small blue">
-        <!-- Copyright -->
```

Comment:
It's okay if the footer would be removed for me, but I think it would be nicer if there is a little space under a document to please the eye.

Reply:
I just looked kind of weird at first, because my homepage was empty. Maybe it looks better with a full homepage.

Reply:
The new footer looks great! Thanks